docs/contributing/features.md

```diff
@@ -1,24 +1,33 @@
 # New Features
 
-We review contributions of new features (e.g., new packages, new scripts, new flags) to the core stacks in an attempt to balance the value of the images to the Jupyter community with the cost of maintaining the images over time. Roughly speaking, we think about the following when considering additions:
+Thank you for contributing to docker-stacks! We review pull requests of new features (e.g., new packages, new scripts, new flags) to balance the value of the images to the Jupyter community with the cost of maintaining the images over time.
+
+## Suggesting a New Feature
```

Comment:
I like the suggest-evaluate-submit workflow. Your earlier comment about drive-by PR still stands. In those cases, I think we can treat the PR as the suggestion, albeit with initial implementation, point to the contributor to this  documentation, and then continue with the evaluation.